var/spack/repos/builtin/packages/miniconda3/package.py

```diff
@@ -13,6 +13,7 @@ class Miniconda3(Package):
     homepage = "https://conda.io/miniconda.html"
     url      = "https://repo.continuum.io/miniconda/Miniconda3-4.6.14-Linux-x86_64.sh"
 
+    version('4.8.2', sha256='5bbb193fd201ebe25f4aeb3c58ba83feced6a25982ef4afa86d5506c3656c142', extension='.sh', url='https://repo.anaconda.com/miniconda/Miniconda3-py38_4.8.2-Linux-x86_64.sh', expand=False)
```

Comment:
`extension` should only be required for compressed archive suffixes that Spack isn't familiar with (but that would be understood by the `tar` program). Did you find you needed it here?

Reply:
A significative suggestion, the Spack‘s identification process was still going smoothly when I tried again.